docs/install.md

```diff
@@ -0,0 +1,130 @@
+Install `hsd`
+=============
+
+`hsd` requires Node.js v12 or higher.
+
+Table of contents
+-----------------
+
+<!-- markdown-toc -i install.md -->
+
+<!-- toc -->
+
+- [Verifying keys](#verifying-keys)
+- [Building From Source (git and npm)](#building-from-source-git-and-npm)
+  * [Verifying](#verifying)
+- [npm](#npm)
+  * [Verifying](#verifying-1)
+- [macOS](#macos)
+  * [Verifying](#verifying-2)
+- [Docker](#docker)
+  * [Building an image](#building-an-image)
+  * [Running a container](#running-a-container)
+  * [Stopping a container](#stopping-a-container)
+
+<!-- tocstop -->
+
+## Verifying keys
+  Every verifying process described below will need project maintainer
+keys. You can get keys following [Security](../SECURITY.md) document.
+
+
+## Building From Source (git and npm)
+```
+git clone --depth 1 --branch v3.0.1 https://github.com/handshake-org/hsd.git
```

Comment:
Yes, it could be scripted. But I was thinking we should use `latest`, `previous` and `life-support` tags to follow the specific versions, so instead we could use `git clone --depth 1 --branch latest https://github.com/handshake-org/hsd.git`

Reply:
Can we move the `latest` tag whenever we release a patch version, etc? If so I like this a lot, we can leave the README's alone, just make sure to also update the instructions to maintainers about how to tag.